[2021-10-10 17:07:30,364] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result 2021-10-10T14:04:43.758814+00:00 [queued]>
[2021-10-10 17:07:30,369] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_group_result 2021-10-10T14:04:43.758814+00:00 [queued]>
[2021-10-10 17:07:30,369] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 17:07:30,369] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-10 17:07:30,369] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-10 17:07:30,376] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): create_entry_group_result> on 2021-10-10T14:04:43.758814+00:00
[2021-10-10 17:07:30,380] {standard_task_runner.py:52} INFO - Started process 16097 to run task
[2021-10-10 17:07:30,385] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'create_entry_group_result', '2021-10-10T14:04:43.758814+00:00', '--job-id', '96', '--pool', 'default_pool', '--raw', '--subdir', '/usr/local/lib/python3.8/dist-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpp3gz7hv4', '--error-file', '/tmp/tmp_m_4hpwf']
[2021-10-10 17:07:30,386] {standard_task_runner.py:77} INFO - Job 96: Subtask create_entry_group_result
[2021-10-10 17:07:30,438] {logging_mixin.py:109} INFO - Running <TaskInstance: example_complex.create_entry_group_result 2021-10-10T14:04:43.758814+00:00 [running]> on host abd
[2021-10-10 17:07:30,457] {taskinstance.py:1252} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_group_result
AIRFLOW_CTX_EXECUTION_DATE=2021-10-10T14:04:43.758814+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-10T14:04:43.758814+00:00
[2021-10-10 17:07:30,457] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-10 17:07:30,457] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo create_entry_group_result']
[2021-10-10 17:07:30,461] {subprocess.py:74} INFO - Output:
[2021-10-10 17:07:30,462] {subprocess.py:78} INFO - create_entry_group_result
[2021-10-10 17:07:30,462] {subprocess.py:82} INFO - Command exited with return code 0
[2021-10-10 17:07:30,473] {taskinstance.py:1212} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_group_result, execution_date=20211010T140443, start_date=20211010T140730, end_date=20211010T140730
[2021-10-10 17:07:30,515] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-10 17:07:30,560] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
